pkg/controller/sync/controller.go

```diff
@@ -435,13 +429,13 @@ func (s *KubeFedSyncController) ensureDeletion(fedResource FederatedResource) ut
 		return util.StatusAllOK
 	}
 
-	annotations := obj.GetAnnotations()
-	orphanResources := annotations != nil && annotations[OrphanManagedResources] == "true"
+	orphanResources := util.IsOrphaningEnabled(obj)
```

Comment:
(No action required) Maybe just inline this call? The intent of the `orphanResources` variable was to ensure the conditional was self-documenting and this could be accomplished with the new method call.

Reply:
Done